e2e/utils.go

```diff
@@ -82,6 +88,112 @@ var (
 	clusterID        string
 )
 
+type cephfsFilesystem struct {
+	Name         string `json:"name"`
+	MetadataPool string `json:"metadata_pool"`
+}
+
+// listCephFSFileSystems list CephFS filesystems in json format.
+func listCephFSFileSystems(f *framework.Framework) ([]cephfsFilesystem, error) {
+	var fsList []cephfsFilesystem
+
+	stdout, stdErr, err := execCommandInToolBoxPod(
+		f,
+		"ceph fs ls --format=json",
+		rookNamespace)
+	if err != nil {
+		return fsList, err
+	}
+	if stdErr != "" {
+		return fsList, fmt.Errorf("error listing fs %v", stdErr)
+	}
+
+	err = json.Unmarshal([]byte(stdout), &fsList)
+	if err != nil {
+		return fsList, err
+	}
+
+	return fsList, nil
+}
+
+// getCephFSMetadataPoolName get CephFS pool name from filesystem name.
+func getCephFSMetadataPoolName(f *framework.Framework, filesystem string, metadataPool *string) {
+	fsList, err := listCephFSFileSystems(f)
+	if err != nil {
+		e2elog.Failf("list CephFS filesystem failed: %v", err)
+	}
+	for _, v := range fsList {
+		if v.Name != filesystem {
+			continue
+		}
+		*metadataPool = v.MetadataPool
+
+		return
+	}
+	e2elog.Failf("metadata pool name not found for filesystem: %s", filesystem)
+}
+
+func compareStdoutWithCount(stdOut string, count int) {
+	stdOut = strings.TrimSuffix(stdOut, "\n")
+	res, err := strconv.Atoi(stdOut)
+	if err != nil {
+		e2elog.Failf("failed to convert string to int: %s", stdOut)
+	}
+	if res != count {
+		e2elog.Logf("expected omap object count %d, got %d", count, res)
```

Comment:
log stdOut here for better debugging.

Reply:
StdOut is just a digit in its string form, which we anyway log after converting it to int as `res`

Reply:
To help to debug we need a way to log the output from the `rados` command.

Reply:
@pkalever can you please check this one? this is really helpful for debugging